kernel/src/hil/nfc.rs

```diff
@@ -0,0 +1,105 @@
+use crate::returncode::ReturnCode;
```

Comment:
Really a comment for line 0: Can you add some top-level documentation (`//!` comments) about what this HIL is for -- especially what parts of the wide array of possible NFC implementations that it does and doesn't cover?